src/components/Profile/Profile.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {useState, useEffect} from 'react';
+import AddFamilyPopup from './AddFamilyPopup';
+import YouAndYourFamilyCard from './YouAndYourFamilyCard';
+import ContactPersonCard from './ContactPersonCard';
+import CarsAndSeatsCard from './CarsAndSeatsCard.js';
+import ContactImage from '../../assets/noun_upload photo_1337310 1.png';
+import EditContactPerson from './EditContactPerson.js';
+import Image from '../../assets/norwegian_fjord.png';
+import API from '../API/API.js';
+
+/** 
+ *  @public Profile displays the different Card types and formats them
+ */
+
+function Profile () {
+    const [showEditContactMember, setShowEditContactMember] = useState(false);
+    const [showAddFamilyPopup,setShowAddFamilyPopup] = useState(false);
+    let contactPersonData = {};
+
+/*     async function fetchData(){
+        API.getContactMember()
+        .then((userObject)=> {
+            contactPersonData = userObject;
+    })}
+
+    useEffect(()=> {
+        await fetchData();
```

Comment:
you can't use await in `useEffect`. better use the `.then` on the Promise instead.